cleanlab/example_models/README.md

```diff
@@ -9,7 +9,7 @@ The dependencies are as follows
 * mnist_pytorch.py - used for training a simplified AlexNet on MNIST using PyTorch
    - pytorch
    - torchvision
-* fastext.py - used for supervised learning on text classification using FastText
-   - fastext
+* fasttext.py - used for supervised learning on text classification using FastText
+   - fasttext
 
-You must install these dependencies on your own if you wish to use them. cleanlab will not force you to install these large deep learning frameworks because that may potentially clobber your code base. For example if you have both TensorFlow and PyTorch installed, issues sometims arise.
+You must install these dependencies on your own if you wish to use them. cleanlab will not force you to install these large deep learning frameworks because that may potentially clobber your code base. For example if you have both TensorFlow and PyTorch installed, issues sometimes arise.
```

Comment:
nit: I think "clobber" isn't the right word to use here.
"may potentially clobber your code base." -> "may potentially install conflicting dependencies."